MonoGame.Framework/Utilities/FileHelpers.cs

```diff
@@ -44,7 +47,7 @@ public static string ResolveRelativePath(string filePath, string relativeFile)
                 filePath = ForwardSlashString + filePath;
 
             // Get a uri for filePath using the file:// schema and no host.
-            var src = new Uri("file://" + filePath);
+            var src = new Uri("file://" + UrlEncode(filePath));
 
             var dst = new Uri(src, relativeFile);

```

Comment:
I would suggest encoding the relative path as well.  It could contain characters in need of encoding.